translations/tiled_de.ts

```diff
@@ -593,17 +593,17 @@
     <message>
         <location line="+8"/>
         <source>&amp;Never</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Nie</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>For &amp;Selected Objects</source>
-        <translation type="unfinished"></translation>
+        <translation>Für ausge&amp;wählte Objekte</translation>
```

Comment:
I think sticking to the &S makes it easier for the people who jump 
between the english and german version. At least so far I tried to stick 
to the english accessor keys.